modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -47,7 +47,9 @@ The following procedure is applicable for already installed and running instance
 +
 [CAUTION]
 ====
-* Certificate files are typically stored as Base64 ASCII files, such as `.pem`, `.crt`, `.ca-bundle`. But, they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64 ASCII certificate rather than the binary-encoded certificate.
+* Certificate files are typically stored as Base64 ASCII files, such as `.pem`, `.crt`, `.ca-bundle`. But they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64 ASCII certificate rather than the binary-encoded certificate.
```

Comment:
```suggestion
* Certificate files are typically stored as Base64 ASCII files, such as files of the `.pem`, `.crt`, and `.ca-bundle` formats. But they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64 ASCII certificate rather than the binary-encoded certificate.
```
Question: 
- Are there more types of file formats other than the ones mentioned here?

Suggestion:
- Can we say: "All Secrets that hold certificate files must use the Base64 ASCII certificate."?

Reply:
@themr0c @svor can you please help me with answering the question above? thanks beforehand!

Reply:
@deerskindoll If there are only 3 formats, then say: "`.pem`, `.crt`, and `.ca-bundle` formats."

If there are more than 3, then say: "in formats such as `.pem`, `.crt`, `.ca-bundle`, and so on.

Reply:
Now I am reading it again and I am not sure. This requires testing: PEM = DER + base64 encoded + header + footer. So we need something inbetween for Java? PEM without header and footer?

Reply:
Let's clarify: these are common formats:
* DER format: binary format.
* PEM format: the base64 encoding of the DER encoding with header and footer lines added. 

The various inputs are conflicting. Can we clarify the required format for the certificate:

* Not DER. Seems acquired.
* PEM. (See: https://github.com/eclipse/che/issues/18352)
* A custom format: PEM, with header and footer lines removed. (See: actual docs content)

Then we have also to adapt the openssl command (add a pipe to remove the headers if required).

Reply:
Hi all, I updated the wording of the disputed line. Please check the updated contents and let me know if they're correct or need additional edits. Thanks!